src/components/productListing/ProductsContainer.js

```diff
@@ -1,7 +1,11 @@
 import React, { useEffect, useState } from "react";
 import axios from "axios";
 import "./ProductsContainer.css";
-
+import { useProductsFilters } from "../../contexts/filter-context";
+import {
+  getSortedProducts,
+  getFilteredProducts,
+} from "../../utilities/filters-utils";
 const ProductsContainer = () => {
```

Comment:
try to give this function in separate file so that it look clean code

Reply:
sure